source/_integrations/meteo_france.markdown

```diff
@@ -5,21 +5,32 @@ logo: meteo-france.png
 ha_release: 0.89
 ha_iot_class: Cloud Polling
 ha_category:
-  - Hub
   - Sensor
   - Weather
 ---
 
 The `meteo_france` integration uses the [Météo-France](http://www.meteofrance.com/) web service as a source for meteorological data for your location. The location is based on the `city` configured in your `configuration.yaml` file.
 
-There is currently support for the following device types within Home Assistant:
+There is currently support for the following platforms within Home Assistant:
 
 - Sensor
 - Weather
 
 It displays the current weather along with a 4 days forecast and can create sensors based on the `monitored_conditions` set in your `configuration.yaml` file, including weather alerts from [Vigilance Météo-France](https://vigilance.meteofrance.com/)
 
-## Configuration
+## Setup the integration
+
+There is two ways to integrate Météo-France in Home Assistant
+
+### Via the frontend
+
+Menu: *Configuration* -> *Integrations*
+
+Search for "Météo-France", add your city, click submit, you are done !
+
+Notice: all `monitored_conditions` will be added, see more details below.
```

Comment:
Not a real documentation review comment; We should remove the `monitored_conditions` altogether.

Reply:
Emmm, wait.

Should I remove this line or `monitored_conditionsˋ should be removed from the parent ?

Reply:
Ideally, it should be removed from the integration, for more information see: <https://github.com/home-assistant/architecture/blob/master/adr/0003-monitor-condition-and-data-selectors.md>

Reply:
Will look at it.

Thanks for the info !